src/Firebase/Database.purs

```diff
@@ -12,3 +13,9 @@ foreign import onValue
    . String
   -> (Json -> Eff (firebase :: FIREBASE | eff) Unit)
   -> Eff (firebase :: FIREBASE | eff) (Eff (firebase :: FIREBASE | eff) Unit)
+
+foreign import write
+  :: ∀ eff
+   . String
+  -> Json
+  -> Eff (firebase :: FIREBASE | eff) Unit
```

Comment:
AFAIK it'll work just fine, but it could throw for all sorts of other reasons, like network or auth issues, or because the specified `path` doesn't exist. Not sure if I should add an `EXCEPTION` effect for that though.